docs/notebooks/working_with_the_ensemble.ipynb

```diff
@@ -1,17 +1,19 @@
 {
  "cells": [
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Working with the lsstseries `Ensemble` object"
+    "# Working with the TAPE `Ensemble` object"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When working with many lightcurves, the lsstseries `Ensemble` object serves as a singular interface for storing, filtering, and analyzing timeseries data. \n",
+    "When working with many lightcurves, the tape `Ensemble` object serves as a singular interface for storing, filtering, and analyzing timeseries data. \n",
```

Comment:
tape is all caps in the rest of the notebooks. Maybe do that here for consistency.